examples/server/public/index.html

```diff
@@ -0,0 +1,459 @@
+            transition: background-color 0.2s;
+        }
+
+        .submit-btn:hover {
+            background-color: #2563eb;
+        }
+
+        .submit-btn:disabled {
+            background-color: #93c5fd;
+            cursor: not-allowed;
+        }
+
+        .spinner {
+            animation: spin 1s linear infinite;
+            margin-right: 8px;
+        }
+
+        @keyframes spin {
+            from { transform: rotate(0deg); }
+            to { transform: rotate(360deg); }
+        }
+
+        .error-message {
+            color: #ef4444;
+            font-size: 0.875rem;
+            text-align: center;
+            margin-top: 15px;
+            display: none;
+        }
+
+        .audio-container {
+            margin-top: 20px;
+            display: none;
+        }
+
+        audio {
+            width: 100%;
+            margin-top: 10px;
+        }
+
+        .audio-footer {
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+            margin-top: 8px;
+            font-size: 0.875rem;
+            color: #6b7280;
+        }
+
+        .download-btn {
+            color: #3b82f6;
+            text-decoration: none;
+            display: flex;
+            align-items: center;
+            cursor: pointer;
+        }
+
+        .download-btn:hover {
+            text-decoration: underline;
+        }
+
+        .hint {
+            font-size: 0.75rem;
+            color: #6b7280;
+            margin-top: 4px;
+        }
+    </style>
+</head>
+<body>
+    <div class="container">
+        <div class="header">
+            <h1>TTS.cpp Server API</h1>
+        </div>
+        
+        <div class="content">
+            <form id="tts-form">
+                <div class="form-group">
+                    <label for="text-input">Text to convert</label>
+                    <textarea id="text-input" placeholder="Enter text to convert to speech..."></textarea>
+                </div>
+                
+                <div class="form-group">
+                    <button type="button" id="advanced-toggle" class="toggle-button">
+                        <span>Advanced Parameters</span>
+                        <svg id="advanced-icon" class="toggle-icon" width="20" height="20" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
+                            <polyline points="6 9 12 15 18 9"></polyline>
+                        </svg>
+                    </button>
+                    
+                    <div id="advanced-parameters" class="advanced-params">
+                        <div class="form-group">
+                            <label for="base-text-url">Base URL</label>
+                            <input type="text" id="base-text-url" placeholder="http://127.0.0.1:8080">
+                            <p class="hint">The address to the TTS server</p>
+                        </div>
```

Comment:
Either:
- Add an API key box here (preferred), or
- Remove the base URL and remove the API_KEY constant and Authentication header from the JS